### DISCUSSION TITLE
OpenChrom Target (Dev) Target Platform is broken

### DISCUSSION THREAD

**Discussion by Mailaender** [OpenChrom/openchrom#484] [Q&A] [answered]
![grafik](https://user-images.githubusercontent.com/756669/65027531-cf413b00-d93a-11e9-87ad-ac23e3f417cd.png)

**Comment by laeubi** [comment, marked as answer]
32bit platform is no longer supported, please use a 64bit JVM to start eclipse/devlop openchrom.

**Comment by Mailaender** [comment]
That's it. Thanks for the quick reply!
